src/main/java/games/strategy/triplea/TripleAPlayer.java

```diff
@@ -222,7 +222,7 @@ private void userActions(final boolean firstRun) {
           + getPlayerBridge().getRemoteDelegate().getClass();
       // for some reason the client is not seeing or getting these errors, so print to err too
       System.err.println(errorContext);
-      ClientLogger.logQuietly(e);
+      ClientLogger.logQuietly(errorContext, e);
```

Comment:
Ditto on the reason given above for not using `ClientLogger#logError()`.  (This comment applies to all similar changes in this file.)